migration-0.45/dashboards.go

```diff
@@ -0,0 +1,130 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"time"
+
+	"github.com/jmoiron/sqlx"
+	_ "github.com/mattn/go-sqlite3"
+)
+
+var (
+	db *sqlx.DB
+)
+
+type Dashboard struct {
+	Id        int       `json:"id" db:"id"`
+	Uuid      string    `json:"uuid" db:"uuid"`
+	CreatedAt time.Time `json:"created_at" db:"created_at"`
+	CreateBy  *string   `json:"created_by" db:"created_by"`
+	UpdatedAt time.Time `json:"updated_at" db:"updated_at"`
+	UpdateBy  *string   `json:"updated_by" db:"updated_by"`
+	Data      string    `json:"data" db:"data"`
+	Locked    *int      `json:"isLocked" db:"locked"`
+}
+type DashboardData struct {
+	Description string						`json:"description"`
+	Tags        []string					`json:"tags"`
+	Name        string						`json:"name"`
+	Layout      []Layout					`json:"layout"`
```

Comment:
![image](https://github.com/SigNoz/signoz-db-migrations/assets/54737045/a4202c4d-796b-4c4f-9e36-db9e3b32285a)
shows fine in my editor 😅

Reply:
loll, it must be combination of tab/whitespace, use whitespaces 😅

Reply:
done 👍